keras/src/backend/torch/layer.py

```diff
@@ -1,39 +1,117 @@
-from typing import Iterator
-from typing import Tuple
-
 import torch
 
-from keras.src.backend.common.stateless_scope import in_stateless_scope
 from keras.src.ops.operation import Operation
 
 
 class TorchLayer(torch.nn.Module):
-    def _post_build(self):
-        # Do not track variables when in a stateless scope.
-        # The variables are not initialized.
-        if in_stateless_scope():
-            return
-        self._track_variables()
-
-    def _track_variables(self):
+    """Adaptation layer to make sure keras.layers.Layer works well with
+    torch.nn.Module. Currently, the main modification are on parameter/module
+    tracking and pointing torch.nn.Module.forward() to the right keras call.
+
+    Module tracking:
```

Comment:
For a section outside of the normal set of sections to be highlighted, use markdown syntax, e.g.

```
**Module tracking:**

(content)
```